### PR TITLE
Pg service test + last bits of sslmode implementation

### PR DESCRIPTION
**Pull request by domi4484**

### CODE REVIEW

File: QgisModelBaker/gui/panel/pg_config_panel.py [hunk 1]
```diff
@@ -20,7 +20,7 @@
 from enum import IntEnum
 
 try:
-    import QgisModelBaker.libs.pgserviceparser
+    import QgisModelBaker.libs.pgserviceparser as pgserviceparser
```
Comment (domi4484) [inline QgisModelBaker/gui/panel/pg_config_panel.py:23]: @signedav not sure why the tests were failing locally without this... does it make sense for you?

Comment (signedav) [reply QgisModelBaker/gui/panel/pg_config_panel.py:23]: locally are not using this import right?

**Review by signedav — APPROVED** [review]

File: QgisModelBaker/gui/panel/pg_config_panel.py (hunk 1 above)
Comment (domi4484) [reply QgisModelBaker/gui/panel/pg_config_panel.py:23]: Thanks for the explanation :)

Comment (signedav) [reply QgisModelBaker/gui/panel/pg_config_panel.py:23]: This should work as well

```
from QgisModelBaker.libs import pgserviceparser
```

**Pull request merged**